scripts/bootstrap.py

```diff
@@ -4,7 +4,7 @@
 This script takes care of fetching, building and installing all WPE Android dependencies,
 including libwpe, WPEBackend-android and WPEWebKit.
 
-The cross-compilation work is done by Cerbero: https://gitlab.igalia.com/ferjm/cerbero
+The cross-compilation work is done by Cerbero: https://github.com/Igalia/cerbero.git
```

Comment:
Should we rename it to wpe-android-cerbero? I'm wondering if we can have more repos using cerbero in the future

Reply:
Well, it is a fork of https://github.com/GStreamer/cerbero and, in theory, once we fix the build for the rest of the architectures we will be contributing the changes upstream, as they will be useful to build [GstWPE](https://gstreamer.freedesktop.org/documentation/wpe/index.html?gi-language=c). So I think it is fine to leave it that way.

Reply:
ok, makes sense